src/drive/web/modules/navigation/Breadcrumb.jsx

```diff
@@ -1,6 +1,6 @@
 /* global __TARGET__ cozy */
 import React, { Component } from 'react'
-import { Link, withRouter } from 'react-router'
+import { withRouter } from 'react-router'
```

Comment:
Same deal with the `Breadcrumb` as with the `File` component — we don't want to be tied to a URL navigation at all cost. This file now exposes "dumb" components that can be used without router, and components with a router attached, that work just like before.